infrastructure/metadata/infrastructure/server/http2-websocket.sub.h2.any.js.ini

```diff
@@ -1,10 +1,10 @@
 [http2-websocket.sub.h2.any.html]
   [WSS over h2]
     expected:
-      if product != "firefox": FAIL
+      if product == "safari" or product == "epiphany" or product == "webkit": FAIL
```

Comment:
This change seems unrelated to the revert? Is it that Chrome was failing previously (for <= 91.0.4449.6), and now passes with a later version? I just wouldn't expect a revert to have other changes.

Reply:
It's because Chrome now passes these tests, so there was an unexpected pass. While it was pinned this went unnoticed.

Reply:
Ok, sounds good.